ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactZIndexedViewGroup.java

```diff
@@ -12,4 +12,10 @@
    * @return The child view index considering z-index
    */
   int getZIndexMappedChildIndex(int index);
+
+  /**
+   * Redraw the view based on updated z-index. This should be called after updating one of it's child
+   * z-index.
+   */
+  void updateZIndex();
```

Comment:
I think that name of this method is a bit too short. Maybe ` updateViewParentZIndex()` would be better as I suppose this method belongs to parent not child.